charts/dis/files/streams/external-ingest.yaml

```diff
@@ -1,25 +1,41 @@
-## Generated by config-gen DO NOT EDIT
 input:
-  label: "ingest_server_input"
-  dimo_http_server:
-      address: 0.0.0.0:9443
-      path: /
-      allowed_verbs:
-        - POST
-      timeout: 5s
-      rate_limit: "connection_rate_limit"
-      tls:
-        enabled: true
-        client_root_cas_file: /etc/ssl/certs/dis/root_ca.crt
-        server_certs:
-          - cert_file: /etc/ssl/certs/dis/tls.crt
-            key_file: /etc/ssl/certs/dis/tls.key
-        require_mutual_tls: true
-      sync_response:
-        last_message_only: true
-        status: ${!meta("response_status").or(200)}
-        headers:
-          Content-Type: application/octet-stream
+  type: broker
+  broker:
+    inputs:
+      - label: "dimo_http_connection_server"
+        dimo_http_connection_server:
+            address: 0.0.0.0:9443
+            path: /
+            allowed_verbs:
+              - POST
+            timeout: 5s
+            rate_limit: "connection_rate_limit"
+            tls:
+              enabled: true
+              client_root_cas_file: /etc/ssl/certs/dis/root_ca.crt
+              server_certs:
+                - cert_file: /etc/ssl/certs/dis/tls.crt
+                  key_file: /etc/ssl/certs/dis/tls.key
+              require_mutual_tls: true
+            sync_response:
+              last_message_only: true
+              status: ${!meta("response_status").or(200)}
+              headers:
+                Content-Type: application/octet-stream
+
+      - label: "dimo_http_attestation_server"
+        dimo_http_attestation_server:
+            address: 0.0.0.0:9442 # TODO(ae): confirm
```

Comment:
Do we need some port/ingress setup to make this work?

Reply:
I think we do... @KevinJoiner not sure what would be required!